support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouSocialShare.jsx

```diff
@@ -32,26 +32,30 @@ const buttonsContainer = css`
   }
 `;
 
-type PropTypes = { email: string, createReferralCodes: boolean, campaignCode: ?string, isUsEndOfYearAppeal: boolean }
+type ContributionThankYouSocialShareProps = {|
+  email: string,
+  createReferralCodes: boolean,
+  campaignCode: ?string
+|};
```

Comment:
I renamed this to be inline with the other TY page components, also we're using flows 'exact' types again (added the pipe characters back)